docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,51 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into our code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
+the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>`.
+
+Before opening pull request
+===========================
+
+Before you create the pull request, you should go through the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>` to ensure
+the proposed changes are required and up to our standards.
+
+If you want to work on the pull request or are not yet finished with the code, please indicate this by marking the pull request as
+`wip <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_.
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
+
+Finally, if you are addressing an existing issue, make sure that issue is linked in your PR. If there is not an existing issue, then you should either create an issue or address WHY you are opening the PR specifically.
+
+During review
+=============
+
+As a reviewer, please follow these rules of thumb:
+
+#. Comments should be clear in addressing why you want to see the change
+#. Comments should be polite, but straightfoward and candid
+#. If you leave a review, continue to follow up on replies to your questions or comments and review the changes you requested
+#. It is polite, but not required, to provide examples for suggestions (particularly for things like name changes)
+
+Before merging
+==============
+
+Before merging, a pull request needs one approving review. While the review is open, anyone can make comments or request changes on the PR.
+
+Although only one approval is required, you must follow these rules:
+
+#. If there is someone with a particular expertise or vested interest in your changes, **do not merge or close the pull request until they get a chance to review.**
```

Comment:
Should we set a rule of thumb for how long you should keep a PR open to give people time to review? Like 5 working days unless it's time sensitive or you get confirmation that no one else is planning to review?

Reply:
I do like that, although I think 5 is a little long. Maybe 3? It would be good to add a note that if you don't get any reviews after 3-4 days, to ping a reminder in the PR or in the slack channel, or ask the person who is closest to the issue/best reviewer.

Reply:
Yeah 3 would be good. I just want to avoid a situation where someone gets a review and approval on the same day they open the PR, then merge it right away before others have a chance to look, so 3 days sounds good too.